calico_cni_k8s_test.go

```diff
@@ -30,6 +32,8 @@ import (
 func init() {
 	// Create a random seed
 	rand.Seed(time.Now().UTC().UnixNano())
+	logrus.SetFormatter(&logutils.Formatter{})
```

Comment:
oh crap, you're right.  My bad didn't read carefully enough.  Since its only a couple of files, I'll fix them up with this PR